### PR TITLE
samd21: gpio support samr21e18a

### PR DESCRIPTION
**Pull request by Hyungsin**
This PR adds CPU_MODEL_SAMR21E18A for GPIO usage.
Some SAMR21 boards (at least mine) give compile error without this fix.

### CODE REVIEW

**Comment by aabadie** [comment]
Are you sure the interrupt mapping is the same for the 2 cpu flavours ?

**Comment by Hyungsin** [comment]
SAMR21 datasheet has the following chart. It seems that SAMR21G and SAMR21E have same configuration in terms of EXTINT.
image

**Comment by Hyungsin** [comment]
@aabadie, oh I don't think so, a great mistake. I will update this to support samr21e :)

**Comment by Hyungsin** [comment]
@aabadie, sorry for the late update, now fixed :)

File: cpu/samd21/periph/gpio.c [hunk 1]
```diff
@@ -52,11 +52,16 @@ static const int8_t exti_config[2][32] = {
       0,  1,  2,  3,  4,  5,  6,  7, 12, 13, -1, 15,  8, -1, 10, 11},
     {-1, -1,  2,  3, -1, -1, -1, -1,  8,  9, 10, 11, -1, -1, -1, -1,
      -1, -1, -1, -1, -1, -1,  6,  7, -1, -1, -1, -1, -1, -1, -1, -1},
-#elif CPU_MODEL_SAMR21G18A
+#elif CPU_MODEL_SAMR21G18A
```
Comment (aabadie) [inline cpu/samd21/periph/gpio.c]: Trailing whitespace: the CI won't like that ;)

**Comment by Hyungsin** [comment]
Thanks! Fixed! :)

**Comment by aabadie** [comment]
and go!

**Comment by Hyungsin** [comment]
Thanks!

**Pull request merged**